crates/storage/src/lazy/mapping.rs

```diff
@@ -369,9 +369,10 @@ const _: () = {
         KeyType: StorageKey + scale_info::TypeInfo + 'static,
     {
         fn layout(_: &Key) -> Layout {
-            Layout::Root(RootLayout::new::<Self, _>(
+            Layout::Root(RootLayout::new::<_>(
```

Comment:
```suggestion
            Layout::Root(RootLayout::new(
```

I believe for a single inferred generic parameter you just omit the `<_>`. Also in other places.

Reply:
lol yeah certainly, thanks!